.github/workflows/build.yml

```diff
@@ -79,3 +79,31 @@ jobs:
             tags: ghcr.io/espressosystems/staking-ui-service:latest
             inputs: ghcr.io/espressosystems/staking-ui-service@${{ needs.build-amd.outputs.digest }},ghcr.io/espressosystems/staking-ui-service@${{ needs.build-arm.outputs.digest }}
             push: ${{ github.event_name != 'pull_request' }}
+  test-demo:
```

Comment:
This feels like a build step we should put a timing bound on. Something like 2x the expected runtime of the workflow